server/app/routes/utils.js

```diff
@@ -1,5 +1,17 @@
 'use strict';
```

Comment:
From glancing at this, it seems you put even testing utils in here. I would encourage this to be broken up into testing utils, authentication utils, etc.